src/core/dev/ib_ctx_handler.cpp

```diff
@@ -73,30 +73,10 @@ ib_ctx_handler::ib_ctx_handler(struct ib_ctx_handler_desc *desc)
     }
 
     m_p_ibv_context = NULL;
```

Comment:
This part is confusing, because `m_p_ibv_context` initialization is hidden in `set_dpcp_adapter()`. Quick look at this method can make impression that m_p_ibv_context remains NULL and NULL dereference is expected further.

Please leave a comment that this field is initialized inside set_dpcp_adapter().

Reply:
Move the initialization to definition.